datacube/cfg/opt.py

```diff
@@ -286,7 +286,10 @@ def get_val_from_environment(self) -> str | None:
         return None
 
 
-def config_options_for_psql_driver(env: "ODCEnvironment"):
+def config_options_for_psql_driver(env: "ODCEnvironment"
+                                  ) -> list[PostgresURLOptionHandler |
+                                            IAMAuthenticationOptionHandler |
+                                            IntOptionHandler]:
```

Comment:
Should just be list[ODCOptionHandler].  Other postgres options maybe added in future.

Reply:
Fixed this.